### ISSUE TITLE
Focus not-obscured understanding document

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#3722]
From the public comments list ([link to the email](https://lists.w3.org/Archives/Public/public-agwg-comments/2024Mar/0000.html)):

> An error in the Intents sections of the W3 WAI notes on Understanding WCAG2.2 for at least one of two criteria has led me to be a bit confused, and wonder what is correct:
> 
> The two criteria in question are:
> 
> 2.4.11 Focus Not Obscured (Minimum) (Level AA) (2.2)
> 
> [https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum](https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FWAI%2FWCAG22%2FUnderstanding%2Ffocus-not-obscured-minimum&data=05%7C02%7Cacampbell%40nomensa.com%7C21a20f2cbaf644c46e4f08dc39c57067%7Cebea4ad6fbbf43bd8449c56e26692c35%7C0%7C0%7C638448768099060316%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C60000%7C%7C%7C&sdata=T0KrWahgPZ3x%2F%2BAi2QYJH29jmHagahYmKYt6vOwev8M%3D&reserved=0)
> 
> 2.4.12 Focus Not Obscured (Enhanced) (Level AAA) (2.2)
> 
> [https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-enhanced](https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FWAI%2FWCAG22%2FUnderstanding%2Ffocus-not-obscured-enhanced&data=05%7C02%7Cacampbell%40nomensa.com%7C21a20f2cbaf644c46e4f08dc39c57067%7Cebea4ad6fbbf43bd8449c56e26692c35%7C0%7C0%7C638448768099071335%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C60000%7C%7C%7C&sdata=09ChSSKO%2FDxE%2BTVcsBXI5VeHxFEsSdJgaLLL6G2UlyQ%3D&reserved=0)
> 
> So what's the problem?
> 
> Well, 2.4.11 says that semi-opaque overlaps may cause a failure of 1.4.11 Non-text Contrast.. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 1.4.11 should be evaluated...
> 
> Meanwhile, 2.4.12 says that semi-opaque overlaps may cause a failure of 2.4.11 Focus Appearance. When a focus indicator can be covered by a semi-opaque component, the the focus indicator should be assessed against 2.4.11. ...
> 
> The error is not the duplicated use of the word ‘the’ – but hey let’s fix that too 😊
> 
> Unfortunately 2.4.11 is not the Focus Appearance criterion. The number and name of that criterion are mixed up. I initially thought that it should have been 2.4.13 Focus Appearance (Level AAA) (2.2), but then I saw that 2.4.11 references 1.4..11 Non-text Contrast in almost identical context.
> 
> Admittedly, both 2.4.13 Focus Appearance (AAA) (2.2) and 1.4.11 Non-text Contrast (AA) (2.1) have similar requirements of the focus indicator. 1.4.11 is more general, requiring the contrast of non-text elements (of which the focus indicator can be described) to have a contrast ratio of at least 3:1. Meanwhile 2.4.13 is specifically about the focus appearance and includes the same contrast requirements.
> 
> My gut feeling was initially that both 2.4.11 and 2.4.12 should reference the same criterion: either 1.4.11 (which as AA level, will be tested more often) or 2.4.13 (which as a AAA level, will sadly be tested less often).
> 
> Alternatively, maybe it's intentional that 2.4.11 as a AA criterion should reference 1.4.11 also as a AA criterion. While 2.4.12, as a AAA criterion should reference 2.4.13 as another AAA criterion, keeping the references within the same level.
> 
> Either way, the current reference of "2.4.11 Focus Appearance" needs correcting, although I suspect the error has been introduced due to a different numbering in earlier draft versions of WCAG 2.2.

**Comment by giacomo-petri** [comment]
I've been working on a draft. 

Since the 2.4.12 Focus Not Obscured (Enhanced) SC falls under AAA criteria, both AA and AAA standards must be addressed to achieve a specific level. Consequently, I've included both 1.4.11 and 2.4.13 (which had been previously included but with an incorrect number) in the Understanding document.

Regarding 2.4.11 Focus Not Obscured (Minimum), which is categorized as a AA SC, the current phrasing doesn't fully align with our intention to include 2.4.13. The Understanding document states:
> While less than 100 percent opacity does not entirely obscure the component, semi-opaque overlaps may result in a failure of 1.4.11 Non-text Contrast. When a focus indicator is potentially covered by a semi-opaque component, the focus indicator's ability to comply with 1.4.11 should be assessed (and met) while the focus indicator is beneath the semi-opaque component.

I believe it's unnecessary to mention 2.4.13 here. However, if we choose to include it, I suggest removing the "(and pass)" label, as it's implicit that meeting the requirement necessitates passing it.

I'm happy to make further edits to the document and incorporate it into the revision if you believe it would be beneficial.
